android/app/src/main/java/org/fossasia/openevent/core/schedule/DayScheduleFragment.java

```diff
@@ -215,7 +177,7 @@ public void onSessionsDownloadDone(SessionDownloadEvent event) {
     @Override
     public void onSaveInstanceState(@NonNull Bundle bundle) {
         if (isAdded() && searchView != null) {
-            bundle.putString(SEARCH, searchText);
+            dayScheduleFragmentViewModel.setSearchText(searchText);
```

Comment:
Yup it does seem to be redundant. I have noticed that LocationsFragment has a very similar method also called onSaveInstanceState though. Is it also redundant there since LocationsFragment also has a ViewModel now?